template-parts/homepage-landing.php

```diff
@@ -29,6 +29,21 @@
     <!-- If we need scrollbar -->
     <div class="swiper-scrollbar"></div>
 </div>
+<!-- Fetch JSON data for dynamic slider numbers -->
```

Comment:
It looks like the script file 'tji/js/tji-datasets-home.js' is loaded on the homepage. If it's no longer needed could you delete it and it's corresponding include in functions.php. If that file is still needed, could we load all homepage js consistently by including your code below within that js include.

Reply:
Could we also the resolve the error that swiper is causing 
![screen shot 2018-10-04 at 3 57 00 pm](https://user-images.githubusercontent.com/3039125/46507644-6971a700-c7ee-11e8-8382-ddbeda74e0b4.png)

Reply:
@LayaTaal curious what your feedback is -- should we be using functions.php to load js files -- or just keep js inline in the template?

Reply:
I'm having issues moving the javascript into tji-datasets-home.js since I can't use the WP method to get the path to the data file. Found more info here: https://stackoverflow.com/questions/28333384/get-template-directory-uri-in-jquery-and-css
For now, I've deleted the outdate javascript in tji-datasets-home.js and kept everything in the php file. @LayaTaal Please advise!

Reply:
I believe that the Swiper console error is WP related as well. But I'm not sure how to resolve. See more here: https://developer.wordpress.org/reference/functions/wp_enqueue_style/

Reply:
I'm cool with the js in page for now. But could you also delete the reference to '/js/tji-datasets-home.js' from the functions.php file

Reply:
@judan I'm not sure how the link (https://developer.wordpress.org/reference/functions/wp_enqueue_style/) is related to swiper errors. Can you please detail. wp_enqueue_style is a method to load a stylesheet. what does that have to do with swiper?

Reply:
@judan I went ahead and updated your branch with the fix. Since we're not using a bundler we only need to use the swiper.min.js file (not the swiper.esm.bundle.js). We also don't need to host these files on our server, we can use the cdn version the developer provides on cloudflare. We also do not need to include the swiper on all pages, just the home page. You'll see these changes pushed to your branch.